acceptance/lib/helper.rb

```diff
@@ -6,22 +6,23 @@ module Reboot
       require 'rubygems' # this is necessary for ruby 1.8
       require 'puppet/version'
       WINDOWS_SHUTDOWN_ABORT = 'cmd /c shutdown /a'
-      if Gem::Version.new(Puppet.version) < Gem::Version.new('3.4.0')
-        WINDOWS_SHUTDOWN_NOT_IN_PROGRESS = 1116 % 256
-      else
-        WINDOWS_SHUTDOWN_NOT_IN_PROGRESS = 1116
-      end
+      # if Gem::Version.new(Puppet.version) < Gem::Version.new('3.4.0')
+      #   WINDOWS_SHUTDOWN_NOT_IN_PROGRESS = 1116 % 256
+      # else
+      #   WINDOWS_SHUTDOWN_NOT_IN_PROGRESS = 1116
+      # end
```

Comment:
Oh yeah. In that case, just remove this section. Also, you can remove lines 6 and 7, as those libraries are only used here.